webapp/locales/en.json

```diff
@@ -76,54 +76,7 @@
     }
   },
   "code-of-conduct": {
-    "consequences": {
```

Comment:
Same same 👆🏼

Reply:
This is a static, localized html page now.
See `/webapp/locales/html/de/code-of-conduct.html`